edx_name_affirmation/tasks.py

```diff
@@ -107,22 +105,10 @@ def proctoring_update_verified_name(
     status,
     full_name,
     profile_name,
-    is_practice_exam,
-    is_proctored,
-    backend_supports_onboarding
 ):
     """
     Celery task for updating a verified name based on a proctoring attempt
     """
-
-    # We only care about updates from onboarding exams, or from non-practice proctored exams with a backend that
-    # does not support onboarding. This is because those two event types are guaranteed to contain verification events,
-    # whereas timed exams and proctored exams with a backend that does support onboarding are not guaranteed
-    is_onboarding_exam = is_practice_exam and is_proctored and backend_supports_onboarding
```

Comment:
Changing celery task input like this will introduce errors on celery processing after deployment. Please refrain from making this signature change.